data/virt_autotest/guest_params_xml_files/slem_5_3_64_kvm_hvm_x86_64.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>slem</guest_os_name>
+    <guest_version>5.3</guest_version>
+    <guest_version_major>5</guest_version_major>
+    <guest_version_minor>3</guest_version_minor>
+    <guest_os_word_length>64</guest_os_word_length>
+    <guest_build></guest_build>
+    <host_hypervisor_uri></host_hypervisor_uri>
+    <host_virt_type>kvm</host_virt_type>
+    <guest_virt_type>hvm</guest_virt_type>
+    <guest_machine_type></guest_machine_type>
+    <guest_arch>x86_64</guest_arch>
+    <guest_name></guest_name>
+    <guest_domain_name></guest_domain_name>
+    <guest_memory>2048,maxmemory=4096</guest_memory>
+    <guest_vcpus>2,maxvcpus=4</guest_vcpus>
+    <guest_cpumodel></guest_cpumodel>
+    <guest_metadata></guest_metadata>
+    <guest_xpath></guest_xpath>
+    <guest_installation_automation>autoyast</guest_installation_automation>
+    <guest_installation_automation_file>slem_5_plus_kvm_hvm_guest_graphical_x86_64.xml</guest_installation_automation_file>
+    <guest_installation_method>location</guest_installation_method>
+    <guest_installation_extra_args>sshd=1#sshpassword=nots3cr3t#console=ttyS0,115200n8#textmode=1#rd.debug#rd.memdebug=4#rd.udev.debug</guest_installation_extra_args>
+    <guest_installation_wait></guest_installation_wait>
+    <guest_installation_method_others></guest_installation_method_others>
+    <guest_installation_media>http://openqa.suse.de/assets/repo/SLE-5.3-Micro-POOL-x86_64-Build12345-Media1/</guest_installation_media>
```

Comment:
Will this media link be replaced with each sle micro 5.3 daily build once we deploy the test on osd?

Reply:
The link in this profile will not be changed. The daily build number will be kept update-to-date by each test run automatically.

Reply:
that's what I ask for, good.